src/coreclr/jit/lsraarm64.cpp

```diff
@@ -1992,9 +1992,13 @@ int LinearScan::BuildHWIntrinsic(GenTreeHWIntrinsic* intrinsicTree, int* pDstCou
                 case NI_Sve_GatherVectorInt32WithByteOffsetsSignExtend:
                 case NI_Sve_GatherVectorSByteSignExtend:
                 case NI_Sve_GatherVectorUInt16WithByteOffsetsZeroExtend:
+                case NI_Sve_GatherVectorUInt16WithByteOffsetsZeroExtendFirstFaulting:
```

Comment:
@TIHan - did you miss adding `GatherVectorFirstFaulting` and `LoadVectorFirstFaulting`? Can you check how is RefPosition for `op2` is created? I guess we are missing for bunch of others too.

Reply:
@mikabl-arm - same for you in https://github.com/dotnet/runtime/pull/104964